locales/de/de.json

```diff
@@ -3,8 +3,6 @@
     "60 Days": "60 Tage",
     "90 Days": "90 Tage",
     ":amount Total": ":amount Gesamt",
-    ":resource Details": ":resource Details",
```

Comment:
Don't delete tags

Reply:
i assumed that excludes should be not be keyed. fixed